i18n/en-US.properties

```diff
@@ -24,5 +24,9 @@ ba.popups.popupHighlight.restrictedPrompt = Comments restricted to single page
 ba.popups.popupList.prompt = Mention someone to notify them
 # Button label for creating a description, comment, or reply
 ba.popups.post = Post
+# Button title for redoing a staged drawing
+ba.popups.redoDrawing = Redo
```

Comment:
Should we start to namespace these a bit more to group them together? `ba.popups.drawing.redo`?